aimm_adapters/heald_labview.py

```diff
@@ -258,14 +258,20 @@ def complete_build_reader(filepath, no_device=False):
         std_df, changed_columns = normalize_dataframe(df, standardize=True)
 
         if std_df is None:
+            element = {"symbol": None, "edge": None}
+            metadata["common"] = {
+                "element": element,
+                "columns": list(df.columns),
+                "specs": ["heald", "experiment", "xas"],
```

Comment:
Since `specs` have special significance to Tiled, I think they should be promoted out of the `metadata` namespace as a peer to `metadata`. Perhaps the same for `columns`.

Reply:
`node.metadata["common"]["specs"]` feels nicer than `node.item["attributes"]["specs"]`. It seems natural to think about the list of specs as a part of the node's metadata. I expect for most of these specs we will not implement any specialized adapter but just treat them as tags. Of course the layout in the response can be different from the layout in the db but not sure if there's a good reason to make them different.